website/pages/partials/builder/scaleway/Config-required.mdx

```diff
@@ -18,7 +18,5 @@
   get the complete list of the accepted image UUID.
 
 - `commercial_type` (string) - The name of the server commercial type:
-  ARM64-128GB, ARM64-16GB, ARM64-2GB, ARM64-32GB, ARM64-4GB,
-  ARM64-64GB, ARM64-8GB, C1, C2L, C2M, C2S, START1-L,
-  START1-M, START1-S, START1-XS, X64-120GB, X64-15GB, X64-30GB,
-  X64-60GB
+  C1, C2L, C2M, C2S, DEV1-S, DEV1-M, DEV1-L, DEV1-XL,
+  GP1-XS, GP1-S, GP1-M, GP1-L, GP1-XL, RENDER-S
```

Comment:
This file is extracted from the code; to change this part you have to change https://github.com/hashicorp/packer/blob/f8e31bff1218dc888b06039a99033a1f758c31be/builder/scaleway/config.go#L48-L53 and then run `make generate`

Reply:
Indeed, thanks for the tips, We'll do that :slightly_smiling_face: